Source/TerminationConditions/termination_condition_bootstrap.cpp

```diff
@@ -0,0 +1,35 @@
+#include "../API/galapagos.h"
+#include "../galapagos_assemblies.h"
+
+#include "fitness_threshold.h"
+
+GALAPAGOS_API void gc_bootstrap() {
+    gc_core lib("Galapagos.dll");
+    lib.register_termination_condition([](termination_condition_metadata* metadata, termination_condition* termination_condition) {
+        fitness_threshold_metadata* dynamic = dynamic_cast<fitness_threshold_metadata*>(metadata);
+        if(dynamic != nullptr) {
+            termination_condition = new fitness_threshold(dynamic);
+            return true;
+        }
+        return false;
+    });
+
+    // get the handle of galapagos core. we should not need to load it should have called gc_bootstrap
```

Comment:
Are we using any of this commented out stuff?

Reply:
No this was the ad hoc implementation of loading exported dll functions. It will go away eventually after I write unit tests for what we currently have.